python/tox.ini

```diff
@@ -50,7 +50,10 @@ commands_pre =
   bedrock-latest: uv pip install -U boto3
   mistralai: uv pip install --reinstall {toxinidir}/instrumentation/openinference-instrumentation-mistralai[test]
   mistralai-latest: uv pip install -U mistralai
-  openai: uv pip install --reinstall {toxinidir}/instrumentation/openinference-instrumentation-openai[test]
+  openai: uv pip uninstall -r test-requirements.txt
```

Comment:
What makes his uninstall necessary?

Reply:
for running locally, because the env is cached